README.md

```diff
@@ -21,27 +21,28 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This is a migration file. This is where the columns in a table are created. It helps avoid hardcoding the schema file. The numbers at the beginning of its name is simply a time stamp.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The router is responsible for retrieving HTTP requests from the browser and matches it to a controller and action. When the controller receives a request, it makes use of the router to determine which routes to take to carry out the needed action.
```

Comment:
👍 the router receives the request, and maps a path and verb to a controller and action